test/services/reply_info_verifier_test.rb

```diff
@@ -16,7 +16,7 @@ def setup
   end
 
   test "v2: generated info is case-insensitive" do
-    assert_nothing_raised ReplyInfoVerifier::InvalidSignature do
+    assert_nothing_raised do
```

Comment:
Form that takes an argument is deprecated, though I think this has the same behavior.